ArcGISRuntimeSDKQt_CppSamples/Geometry/NearestVertex/NearestVertex.cpp

```diff
@@ -119,7 +119,7 @@ void NearestVertex::setupGraphics()
   graphicsOverlay->graphics()->append(nearestVertexGraphic);
 
   // add graphic to clicked location
-  connect(m_mapView, &MapQuickView::mouseClicked, this, [=](QMouseEvent& e)
+  connect(m_mapView, &MapQuickView::mouseClicked, this, [nearestVertexGraphic, nearestCoordinateGraphic, polygonBuilder, clickedLocationGraphic, this](QMouseEvent& e)
```

Comment:
Please wrap this over a few lines so it's not so long.

Reply:
This is done